Slim/Plugin/WiMP/ProtocolHandler.pm

```diff
@@ -237,8 +237,19 @@ sub _gotTrack {
 
 	my $cache = Slim::Utils::Cache->new;
 	$cache->set( 'wimp_meta_' . $info->{id}, $meta, 86400 );
+	
+	my $http = Slim::Networking::Async::HTTP->new;
+	$http->send_request( {
+		request     => HTTP::Request->new( GET => $info->{url} ),
+		onStream  	=> \&Slim::Utils::Scanner::Remote::parseMp4Header,
+		onError     => sub {
+				my ($self, $error) = @_;
+					$log->warn( "could not find MP4 header $error" );
+					$params->{successCb}->();
+				},
+		passthrough => [ $song->track, { cb => $params->{successCb} }, $info->{url} ],			
+	} );
```

Comment:
This should be conditional, or not spit out a warning? Because this will be run with FLAC files, too?

Reply:
totally, that was just for the quick test :-)